docs/developer_handbook.md

```diff
@@ -89,12 +92,26 @@ our tests will fail if you don't provide this support. Note that in step 3 below
 4. Document this new filter option in the dist folder's [broker config templates](../dist/src/main/config/).
 5. Add a mapping for the ValueType to the [TestSupport](../exporters/opensearch-exporter/src/test/java/io/camunda/zeebe/exporter/opensearch/TestSupport.java).
 
-### Extend Official documentation
+### Extend official documentation
 
 Our Exporter configurations are documented in the [official docs](https://github.com/camunda/camunda-platform-docs).
 In the previous steps we've extended the configuration of the Elasticsearch and OpenSearch exporters.
 These configurations options need to be added to the official documentation.
 
+### Extend Zeebe Process Test
+
+The [Zeebe Process Test (ZPT)](https://github.com/camunda/zeebe-process-test/) library allows you to unit test your Camunda Platform 8 BPMN processes.
+The [RecordStreamLogger](https://github.com/camunda/zeebe-process-test/blob/main/filters/src/main/java/io/camunda/zeebe/process/test/filters/logger/RecordStreamLogger.java) class
+defines how the supported `ValueType`s' records are logged in tests.
+
+In the previous steps, we've added a new value type. This type needs to be added here to the `valueTypeLoggers`.
+
+### Add ValueType to supported types for replay
+
+The engine defines a [range of supported value types](https://github.com/camunda/zeebe/blob/main/engine/src/main/java/io/camunda/zeebe/engine/Engine.java#L48-L49) it processes in replay mode.
+In the previous steps, we've added a new value type. This type needs to be added here to ensure that events of this type will be replayed as well. Otherwise, brokers that replay the events
+will miss the entity updates that happened in those events for the new value type which leads to data loss.
+
```

Comment:
This range is used both for processing and replay. It defines the value types that the engine supports. The streamplatform uses this to determine whether the Engine should process/replay records of this type, or that another record processor must handle it.

With processing, your tests will fail if you forgot to add the value type because the engine doesn't process your command then. But, with replay this is less visible, because the engine can apply the events when it initially appends them, but not replay them without being configured as supported.

❌ We should change this text to fit both processing/replay scenarios.

Reply:
Ah nice, I actually wasn't sure about that part! Thanks, @korthout, I'll adjust this and merge it 👍